api/v1/openapi.yaml

```diff
@@ -47,6 +47,12 @@ paths:
         in: header
         required: false
         type: boolean
+      - name: include-k8s-check
```

Comment:
nit: For clarity, I might suggest naming this a little bit more explicitly, something like:

```suggestion
      - name: require-k8s-connectivity
```